ckanext/xloader/config_declaration.yaml

```diff
@@ -2,6 +2,12 @@ version: 1
 groups:
   - annotation: ckanext-xloader settings
     options:
+      - key: ckanext.xloader.site_url
+        default:
```

Comment:
I think we don't need an empty string default, it's normal for optional settings to be not present if not provided
```suggestion
```

Reply:
instead of the `toolkit.config.get("ckanext.xloader.site_url") or toolkit.config.get("ckan.site_url")` logic below we should be able to use just `toolkit.config.get("ckanext.xloader.site_url")` along with:
```
       validators: configured_default("ckan.site_url",None)
```

Reply:
@wardi ok I see. so that default is just set in the config yaml, and then, any code can assume that fallback default for the `ckanext.xloader.site_url` setting?

Reply:
yeah, this validator will set `ckanext.xloader.site_url` to the same as `ckan.site_url` when it's not given. Another one of @smotornyuk 's clever ideas.